api/capi/doc/nnstreamer_doc.h

```diff
@@ -43,6 +43,9 @@
  *
  *  Note that this function set is supposed to be thread-safe.
  *
+ * @subsection CAPI_MEDIA_RECORDER_MODULE_PIPELINE_STATE_DIAGRAM Pipeline State Diagram
```

Comment:
Question : ```CAPI_MEDIA_RECORDER_MODULE``` is it correct?

Reply:
Sorry. I made a mistake. `CAPI_ML_NNSTREAMER_PIPELINE_STATE_DIAGRAM` is right.